src/main/resources/application.yaml

```diff
@@ -53,3 +53,5 @@ idam:
     url: ${S2S_URL}
 
 # end of clients region
+notifications:
+  wait-duration-to-process-pending-in-minute: ${WAIT_DURATION_TO_PROCESS_PENDING_NOTIFICATION_IN_MINUTE}
```

Comment:
can we move/squash to `scheduling.task.pending-notifications` instead? these names are getting ridiculously too long. then it could start with smth like `process-delay-...`

Reply:
can be done.
I did it separate as I thought it is different from scheduling, I may confuse as scheduling as delay as well.